scripts/msg.h.template

```diff
@@ -114,6 +114,13 @@ for constant in spec.constants:
 
 @# Integer constants
 @[if len(constants_integer) > 0]@
+// reducing the odds to have name collisions with Windows.h 
+@[for constant in constants_integer]@
```

Comment:
Why is this only done for some constants - namely integer? I would expect the same chance for non-integer constant.

Reply:
@dirk-thomas You are right. After taking closer look, I agreed that the same mitigation should be applicable to all types of constants.